pkg/datapath/config.go

```diff
@@ -18,8 +18,10 @@ import (
 	"io"
 
 	"github.com/cilium/cilium/pkg/addressing"
+	"github.com/cilium/cilium/pkg/cidr"
 	"github.com/cilium/cilium/pkg/identity"
 	"github.com/cilium/cilium/pkg/mac"
+	"github.com/cilium/cilium/pkg/node"
```

Comment:
I have this sneaking suspicion we're trying to avoid `node` dependency in the `pkg/datapath` so it's mainly a types package.... but I guess if it's a problem, we can always figure out how to rearrange it.

Reply:
While we could remove this import by pushing the implementation to `pkg/node`, I don't think it makes much logical sense there given that this is about how we choose to SNAT or not. I have no better immediate suggestion than keeping the current version.